patches/master_patch.patch

```diff
@@ -921,41 +951,22 @@ index c3f648f165483a2d618438c1a4ef1b35ad6fb10c..3c95e1256465b7b6a009d3bf7619c8f4
    FRIEND_TEST_ALL_PREFIXES(FirefoxImporterTest, ImportBookmarksV25);
    void ImportBookmarks();
 diff --git a/chrome/utility/profile_import_handler.h b/chrome/utility/profile_import_handler.h
-index 257467a99132986f56d3ce098515afab24db7941..d8135a059b8621dbe925574b806452211b076a9c 100644
+index 58d7d47f379b145d5eec6f711cc882430cd1db29..9867e255717e7f2a17c7b5d9b8f0627753a9ae9f 100644
 --- a/chrome/utility/profile_import_handler.h
 +++ b/chrome/utility/profile_import_handler.h
-@@ -38,6 +38,7 @@ class ProfileImportHandler : public chrome::mojom::ProfileImport {
-                      chrome::mojom::ProfileImportRequest request);
+@@ -36,6 +36,7 @@ class ProfileImportHandler : public chrome::mojom::ProfileImport {
+   static void Create(chrome::mojom::ProfileImportRequest request);
  
   private:
 +  friend class BraveProfileImportHandler;
    // chrome::mojom::ProfileImport:
    void StartImport(const importer::SourceProfile& source_profile,
                     uint16_t items,
-diff --git a/components/guest_view/browser/guest_view_base.cc b/components/guest_view/browser/guest_view_base.cc
-index 27570202e8e5cca632e560fe5b22920aeb2c5742..3ac3b8c7b81c095b3b2264ff18431852de060a1b 100644
---- a/components/guest_view/browser/guest_view_base.cc
-+++ b/components/guest_view/browser/guest_view_base.cc
-@@ -432,8 +432,13 @@ void GuestViewBase::DidDetach() {
-   owner_web_contents()->Send(new GuestViewMsg_GuestDetached(
-       element_instance_id_));
-   element_instance_id_ = kInstanceIDNone;
--  if (!CanRunInDetachedState())
```

Comment:
I don't think this is right. This is critical functionality for tab drag and drop cc @bbondy

Reply:
Indeed. It was a chunk that slipped through the cracks, we have fixed that and will be updating the PR including it.